modules/consentManagementUsp.js

```diff
@@ -289,6 +289,7 @@ function exitModule(errMsg, hookConfig, extraArgs) {
 export function resetConsentData() {
   consentData = undefined;
   consentAPI = undefined;
+  consentTimeout = undefined;
```

Comment:
this one stumped me, hat tip to @dgirardi for diagnosing why my 'should disable' test was failing.